### PR TITLE
Noblesse section

### PR DESCRIPTION
**Pull request by bogdanascarlat10**
image

### CODE REVIEW

**Comment by bdogaru** [comment]
Vezi daca poti adauga si un buton care sa trimita catre linkul oficial te rog.

**Comment by bogdanascarlat10** [comment]
Added button and changed icons

File: src/components/general-components/image-text-simple.js [hunk 1]
```diff
@@ -30,14 +30,17 @@ export default function ImageWithText({ image, title, textContent, numbersConten
...
-										<span>{item.title}</span>
+									    <span>{item.image}</span>
+										<span><strong>{item.title}</strong></span>
 										<span>{item.subtitle}</span>
 									</div>
 								</Fade>
 							))}
 						</div>
 					)}
+					{children}
```
Comment (bdogaru) [inline src/components/general-components/image-text-simple.js:41]: Nice approach!

File: src/pages/index.js [hunk 2]
```diff
@@ -87,32 +111,67 @@ export default function Homepage({ data }) {
...
+								image: <GatsbyImage image={lightningIcon} alt="Mobile icon" style={{ width: 30, height: 30 }} />,
+								subtitle: 'Lightweight and fast',
+							},
+							{
+								image: <GatsbyImage image={cartIcon} alt="Mobile icon" style={{ width: 30, height: 30 }} />,
+								subtitle: 'Mobile first checkout',
+							},
+						]}
+					>
+						<Link href="/blog/vevol-themes-first-shopify-theme" className="vm-button vm-button--white mt-5">
```
Comment (bdogaru) [inline src/pages/index.js:154]: Schimba te rog cu URL-ul asta: https://themes.shopify.com/themes/noblesse

Pune-l si cu target="_blank" ca sa se deschida in tab nou. 

Also, schimba mt-5 cu mt-6

File: src/pages/index.js [hunk 3]
```diff
@@ -87,32 +111,67 @@ export default function Homepage({ data }) {
...
+								We're very happy to launch Noblesse, our first Shopify Theme, perfectly adapted to fashion, jewellery and beauty!
 							</p>,
 							<p className="mt-5">
-								Our ethos is to provide quality above all else and we uphold our beliefs when offering
-								our services to our clients.
+								We wanted to create something different, for brands that want to stand out without the need for custom coding, but
+								still CRO oriented.
 							</p>,
+						]}
+						numbersContent={[
```
Comment (bdogaru) [inline src/pages/index.js:139]: Actualizeaza, te rog, iconitele si textul cu ce e mai jos

"Visual Storytelling" - https://www.flaticon.com/free-icon/beautiful_5948580?term=beautiful&page=1&position=4&origin=search&related_id=5948580
"Highly Customisable" - https://www.flaticon.com/free-icon/pallet_5949171?related_id=5949171&origin=pack
"Focused on Conversion" - https://www.flaticon.com/free-icon/bag_5948550?related_id=5948550&origin=pack

**Pull request merged**